pages/docs/manual/latest/exception.mdx

```diff
@@ -6,9 +6,36 @@ canonical: "/docs/manual/latest/exception"
 
 # Exception
 
-Exceptions are just a special kind of variant, thrown in **exceptional** cases (don't abuse them!).
+Exceptions are just a special kind of variant, thrown in **exceptional** cases (don't abuse them!). Consider using the `option` or `result` type for recoverable errors.
```

Comment:
Link to the corresponding docs page(s) for Option and Result please.

Reply:
we don't have documented the `result` builtin type

Reply:
maybe add `Result` page after `Null, Undefined and Option`

Reply:
Added link to option https://github.com/rescript-association/rescript-lang.org/pull/880/commits/8544ec9f27672d1c67d210d3bebff5fe2cd7842e

Reply:
Thanks. Yes we need to document Result somewhere, but we can fix the link afterwards.
Or just link to the Core Result Api page for now?

Reply:
1df7faf3